css/sateesh.css

```diff
@@ -0,0 +1,59 @@
+.cardrow{
+  border-spacing:5px;  
+  margin:0px;
+  
+  
+}
+  
+  
+  
+  .cards { box-shadow: 2px 2px black, 0 6px 20px 0 rgb(255, 255, 255);
```

Comment:
Please do not use predefined class names from bootstrap. Change these class names and add those class names back to the code. Similar issues will be addressed as IBC (Identical bootstrap class)